pep-0615.rst

```diff
@@ -359,7 +395,9 @@ search path at runtime.
 
 .. code-block::
 
-    def set_tzpath(tzpaths: Optional[Sequence[Union[str, Pathlike]]]) -> None:
+    def set_tzpath(
+        tzpaths: Optional[Sequence[Union[str, Pathlike]]] = None
```

Comment:
nit: this should be `PathLike` (or even `os.PathLike` to be explicit)

Reply:
Thanks!